.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,9 @@
 #### Checklist
-- [ ] Unit tests
-- [ ] Updated testplan
+- [ ] Functionality meets specifications
+- [ ] Functionality working
+- [ ] Code reviewed
+- [ ] Code tested
+- [ ] User documentation updated
+- [ ] Test plan template updated
+- [ ] Clean commits
+- [ ] Build passes ('All checks have passed' in pull request)
```

Comment:
This is also specified by a very big green √ at the bottom of a PR.
I don't think this needs to be in the template